packages/legend-query/src/stores/QueryBuilderLambdaBuilder.ts

```diff
@@ -99,6 +101,16 @@ export const buildLambdaFunction = (
 
   // build getAll()
   const getAllFunction = buildGetAllFunction(_class, multiplicityOne);
+  if (
+    _class instanceof Class &&
```

Comment:
Don't think `_class instanceof Clas` is necessary. the `_class` was already checked for nullable so it can only be of type `Class` at this point